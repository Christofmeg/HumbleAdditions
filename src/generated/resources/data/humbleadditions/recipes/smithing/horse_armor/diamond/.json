{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:diamond_block"
  },
  "base": {
    "item": "minecraft:golden_horse_armor"
  },
  "result": {
    "item": "minecraft:diamond_horse_armor"
  }
}