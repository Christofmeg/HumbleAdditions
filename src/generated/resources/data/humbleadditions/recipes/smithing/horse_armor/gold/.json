{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:gold_block"
  },
  "base": {
    "item": "minecraft:iron_horse_armor"
  },
  "result": {
    "item": "minecraft:golden_horse_armor"
  }
}