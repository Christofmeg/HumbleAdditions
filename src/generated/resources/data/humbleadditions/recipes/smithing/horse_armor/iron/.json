{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:iron_ingot"
  },
  "base": {
    "item": "minecraft:leather_horse_armor"
  },
  "result": {
    "item": "minecraft:iron_horse_armor"
  }
}