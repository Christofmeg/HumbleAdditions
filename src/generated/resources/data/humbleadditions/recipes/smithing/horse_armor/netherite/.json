{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:netherite_block"
  },
  "base": {
    "item": "minecraft:diamond_horse_armor"
  },
  "result": {
    "item": "humbleadditions:netherite_horse_armor"
  }
}