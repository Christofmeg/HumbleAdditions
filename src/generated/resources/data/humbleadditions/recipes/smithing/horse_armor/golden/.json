{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:gold_ingot"
  },
  "base": {
    "item": "minecraft:iron_horse_armor"
  },
  "result": {
    "item": "minecraft:golden_horse_armor"
  }
}